packages/components/src/Components/TagCount/TagCount.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './tagCount.scss';
+import { Button } from '@patternfly/react-core';
+import { TagIcon } from '@patternfly/react-icons'
 
 const TagCount = ({count, onTagClick, className, ...props}) => {
     return (
-        <button {...props} className={`ins-c-tag-count ${className}`} onClick={onTagClick}>
-            <i className="fas fa-tag"></i>
+        <Button  {...props} variant="plain" className={`ins-c-tag-count ${className}`} onClick={onTagClick}>
+            <TagIcon />
             <span>{count}</span>
```

Comment:
let's add classes here so we don't style on actual tags
the icon should be something like
`ins-c-tag-count__icon`

and the actual count:
`ins-c-tag-count__text`